apps/reference/docs/guides/cli/custom-hostnames.mdx

```diff
@@ -0,0 +1,75 @@
+---
+id: custom-hostnames
+title: Custom Hostnames
+description: Configuring a Custom Hostname for your Supabase project.
+---
+
+import Tabs from '@theme/Tabs'
+import TabItem from '@theme/TabItem'
+
+:::note
+
+Custom Hostnames are currently in beta, and are being slowly made available to projects.
+
+:::
+
+:::note
+
+The commands in this guide require Supabase CLI 1.10.1+
+
+:::
+
+## Overview
+
+Custom hostnames allow you to present a branded experience to your users.
```

Comment:
Would be useful to have some more context here (e.g., examples of how this shows up for end-users).